src/nfa/concatenation.cc

```diff
@@ -76,23 +76,23 @@ Nfa Algorithms::concatenate_eps(const Nfa& lhs, const Nfa& rhs, const Symbol& ep
     const unsigned long lhs_states_num{lhs.size() };
     const unsigned long rhs_states_num{rhs.size() };
     Nfa result{}; // Concatenated automaton.
-    StateToStateMap lhs_result_states_map_internal{}; // Map mapping rhs states to result states.
-    StateToStateMap rhs_result_states_map_internal{}; // Map mapping rhs states to result states.
+    StateRenaming lhs_result_states_renaming_internal{}; // Map mapping rhs states to result states.
```

Comment:
```suggestion
    StateRenaming _lhs_states_renaming{}; // Map mapping rhs states to result states.
```
How about this? I feel that `_` is ideal synonym for `internal`

Reply:
I will rename the variables. It is shorter yet still clear.